go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/vishvananda/netlink
 go 1.12
 
 require (
-	github.com/vishvananda/netns v0.0.0-20200728191858-db3c7e526aae
-	golang.org/x/sys v0.0.0-20220804214406-8e32c043e418
+	github.com/vishvananda/netns v0.0.1
+	golang.org/x/sys v0.2.0
```

Comment:
This set of gomod changes look unrelated.
Can you push them through a separate PR?

Reply:
Sorry.
Done

Reply:
@aboch 👋